cfme/v2v/migrations.py

```diff
@@ -815,6 +831,19 @@ def step(self):
         self.prerequisite_view.create_infrastructure_mapping.click()
 
 
+@navigator.register(InfrastructureMapping, 'Edit')
+class EditInfrastructureMapping(CFMENavigateStep):
+    VIEW = EditInfrastructureMappingView
+
+    prerequisite = NavigateToAttribute('parent', 'All')
+
+    def step(self):
+        try:
+            self.prerequisite_view.infra_mapping_list.edit_mapping(self.obj.name)
+        except NoSuchElementException:
```

Comment:
Not quite - edit_mapping is already catching `NoSuchElementException` and returning a boolean - here we need to check against the boolean return and raise.